CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 3.4)
 
+configure_file("version.config" "version_config" @ONLY)
```

Comment:
Is this just to trigger CMake when version is updated? Shouldn't this be `COPYONLY`, or does that not trigger a reconfiguration? Why change the file name? Is it to support in-tree builds? I would just create a hidder (dot file) version of this instead, maybe called `.version.config.build` or something?